.pipelines/pipeline.yaml

```diff
@@ -353,20 +353,56 @@ stages:
       clusterName: "swifte2e"
       vmSize: Standard_B2s
       k8sVersion: ""
-      dependsOn: 'containerize'
+      dependsOn: "containerize"
 
   - template: singletenancy/aks/e2e-job-template.yaml
     parameters:
       name: "aks_ubuntu_22_linux_e2e"
       displayName: AKS Ubuntu 22
-      arch: 'amd64'
-      os: 'linux'
+      arch: "amd64"
+      os: "linux"
       clusterType: linux-cniv1-up
-      clusterName: 'ubuntu22e2e'
+      clusterName: "ubuntu22e2e"
       vmSize: Standard_B2s
       k8sVersion: 1.25
       scaleup: 100
-      dependsOn: 'containerize'
+      dependsOn: "containerize"
+
+  - template: singletenancy/cilium/cilium-e2e-job-template.yaml
+    parameters:
+      name: "cilium_podsubnet_cni_e2e"
+      displayName: CNI - Cilium Podsubnet
+      testDropgz: true
+      clusterType: cilium-podsubnet-up
+      clusterName: "ciliumpodsubnetcnie2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: "containerize"
+
+  - template: singletenancy/cilium-overlay/cilium-overlay-e2e-job-template.yaml
+    parameters:
+      name: "cilium_overlay_cni_e2e"
+      displayName: CNI - Cilium Overlay
+      testDropgz: true
+      clusterType: cilium-overlay-up
+      clusterName: "ciliumoverlaycnie2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: "containerize"
+
+  - template: singletenancy/aks-swift/e2e-job-template.yaml
+    parameters:
+      name: "aks_swift_cni_e2e"
+      displayName: CNI - AKS Swift
+      testDropgz: true
+      clusterType: swift-byocni-up
+      clusterName: "swiftcnie2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: "containerize"
```

Comment:
nit: Lets move these next to the appropriate counterpart that does not have `testDropgz: true`